lib/controller.js

```diff
@@ -10,255 +10,260 @@ var updates = require('./update-fetch');
 var inquirer = require('inquirer');
 var colors = require('colors');
 var util = require('util');
+var provision = require('./tessel/provision');
 var controller = {};
 var responses = {
   noAuth: 'No Authorized Tessels Found.',
   auth: 'No Tessels Found.'
 };
 
-
 Tessel.list = function(opts) {
+  return provision.setDefaultKey(opts.key)
```

Comment:
It looks like there are a ton of changes to this file but in reality, I just had to indent most of the contents of the `list` and `get` file after adding:
```
return provision.setDefaultKey(opts.key)
.then(function() {
  ...
}
```